wai-app-static/WaiAppStatic/Storage/Embedded/Runtime.hs

```diff
@@ -15,8 +16,13 @@ import Data.Function (on)
 import qualified Data.Text as T
 import Data.Ord
 import qualified Data.ByteString as S
+#ifdef USE_CRYPTONITE
```

Comment:
I think you could use the build in Cabal macros for this instead. There's definitely `MIN_VERSION_cryptonite`, but I think there may be others too.

Reply:
`MIN_VERSION_cryptonite` seems to work perfectly here, so I have force-pushed revised commits that use this instead. Thank you for the review.